neutron-org/neutron-integration-tests
# #NTRN-347 Removing artifacts from repo

1. Artifacts are removed and `*.wasm` is added to `.gitignore`
2. Added script to download artifacts from Google Cloud bucket
3. Got rid of the artifacts duplication
4. Minor init shell scripts improvements

Before testing don't forget to update yarn dependencies

How to test:
1. update yarn dependencies according to readme
2. download artifacts according to readme
3. run tests as usual, make sure they passed

You can also check different scenarios of using the script besides default one (help, verbose mode, different branch/directory)
